### PR TITLE
Add vip-pools setting

### PR DESCRIPTION
**Pull request by n313893254**
Add vip-pools setting

- https://github.com/harvester/harvester/issues/3100

### CODE REVIEW

File: docs/advanced/settings.md [hunk 1]
```diff
@@ -369,6 +369,21 @@ Default: `https://harvester-upgrade-responder.rancher.io/v1/checkupgrade`
 https://your.upgrade.checker-url/v99/checkupgrade
 ```
 
+## `vip-pools`
+
+This setting allows you to configure the global or namespace IP address pools of the VIP by CIDR.
```
Comment (futuretea) [inline docs/advanced/settings.md:374]:
```suggestion
This setting allows you to configure the global or namespace IP address pools of the VIP by CIDR or IP range.
```

Comment (n313893254) [reply docs/advanced/settings.md]: Done

**Review by futuretea — APPROVED** [review]
LGTM, thanks!

**Review by guangbochen — COMMENTED** [review]
Can you also please help to add it to the cn version, thanks.

File: i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md [hunk 2]
```diff
@@ -365,6 +365,16 @@ false
...
+## `vip-pools`
+
+使用 CIDR 或 IP 范围配置 VIP 的全局或命名空间 IP 地址池。
+
+Default: `{}`
+
+:::note
+从用户界面配置多CIDR或IP范围，只在Harvester v1.1.1中可用。
+:::
+
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md:377]: and examples?

Comment (n313893254) [reply i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md:377]: Done

**Review by guangbochen — APPROVED** [review]
LGTM, and will merge it with the v1.1.1 release, thanks.

**Pull request merged**